src/hooks/useGetQuestion.ts

```diff
@@ -0,0 +1,28 @@
+import { useAppDispatch, useAppSelector } from "../state/reduxHooks";
+import axios from "axios";
+import { useEffect } from "react";
+import { API_URL } from "../settings";
+import { submitQuestion } from "../state/questionSlice";
+
+export function useGetQuestion() {
```

Comment:
We can put this hook in the file `QuestionPage` so that it runs there. 
If you want to test it out, you can comment out the lines that cause it to return prematurely, that way it should run the first time that you open the page.

Reply:
Don't think this has been addressed?

Reply:
this uses `UsePollForGetQuestion` instead

Reply:
This file is no longer being used so it got deleted :)